health_probe.py

```diff
@@ -22,12 +22,12 @@ def broadcast(self, message, service_name, status):
             self.telegram_handler.broadcast('Meu serviço principal foi restaurado. Estou de volta!')
         else:
             print('Ada is ded. Sending telegram message...')
-            self.telegram_handler.broadcast(f"I don't feel so good... Acho que eu morri, estou rodando como backup")
+            self.telegram_handler.broadcast(f"I don't feel so good... estou rodando como backup", sticker=STICKER_ID_KK_MORRI)
 
 
 if __name__ == '__main__':
     logger = Logger(None)
-    handler = TelegramHandler(TELEGRAM_API_TOKEN, CONVERSATIONS)
+    handler = TelegramHandler(logger, TELEGRAM_API_TOKEN, CONVERSATIONS)
     backup_handler = BackupHandler(handler)
-    start_health_check(backup_handler, {'Ada': ADA_URL}, HEALTHCHECK_SLEEP_AMOUNT, HEALTHCHECK_ANOMALY_THRESHOLD)
+    start_health_check(logger, backup_handler, {'Ada': ADA_URL}, HEALTHCHECK_SLEEP_AMOUNT, HEALTHCHECK_ANOMALY_THRESHOLD)
```

Comment:
Since you're passing the logger around, might as well pass the logger to the `BackupHandler` above, and log instead of `print`.